beacon-chain/sync/querier.go

```diff
@@ -71,6 +92,27 @@ func (q *Querier) Stop() error {
 	return nil
 }
 
+func (q *Querier) listenForChainStart() {
```

Comment:
Do we have tests for `listenForChainStart`? 

1.) q.isChainStart  and 2.) err/ctx.done

Reply:
Done ! Added in tests for both of them

Reply:
Thank you!